keyboards/planck/keymaps/lxol/keymap.c

```diff
@@ -0,0 +1,311 @@
+#include "planck.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+enum planck_layers {
+  _QWERTY,
+  _LEFT,
+  _RAISE,
+  _DOUBLERAISE,
+  _FUN,
+  _SYM,
+  _MODIFIER,
+  _MOUSE
+};
+
+#undef LEADERS_REFERENCE_LAYER
+#define LEADERS_REFERENCE_LAYER  _QWERTY
+
+enum planck_keycodes {
+  LEFT = SAFE_RANGE,
+  RAISE,
+  FUN,
+  MOUSE,
+  BACKLIT,
+  LD_RAISE,
+
+  SEQ_SYMBOLS,
```

Comment:
Do these need to consume keycodes? I'm not entirely sure I understand why they're not in their own enum.